docs/development_suite/company/create.mdx

```diff
@@ -7,228 +7,543 @@ sidebar_label: Create a Company
+  - **baseUrl**: base path where the variables will be read from. 
 
-  3. **complete configuration**, for example:
+    If not specified, variables will be read from the first parent group of the Configurations project in GitLab.
 
-     ```json
-     {
-       "type": "gitlab",
-       "baseUrl": "https://my-gitlab-host",
-       "storage": {
-         "type": "projects",
-         "path": "clients/mia-platform/configurations"
-       }
-     }
-     ```
+    As an example, if your GitLab project is saved in **`https://git.tools.mia-platform.eu/mia-platform/Test/configurations`**, the environment variables will be written in the **`mia-platform/Test`** group.
 
-     This configuration is saved only within the company, and should be retrieved at runtime in project fetching from company info. So if your company has all environment variables of the projects set in the parent group, this setting might be changed for all projects at the same time.
+  - **storage**: additional settings related to the storage of environment variables within the GitLab installation.
 
-- `logicalScopeLayers`: an array of objects identifying the set of logical layers available in the current company. Each object is shaped as follows:
+    It is composed by the following properties:
+
+    - **id**: TODO
+
+    - **type**: TODO
+
+    - **path**: Either one of the following:
+
+      - `groups`: TODO
+      - `projects`: TODO
+
+    <br/>
+    
+    :::info
+    This configuration is saved only within the company model, and is retrieved by projects at runtime. 
+
+    If in your company all the environment variables are set to be located in the parent group, by editing this field you can safely change the variables path for all the projects in the company in one go.
+    :::
+
+  :::
+
+  ##### Example Configuration
+  ```json
+  {
+    "providerId": "my-gitlab",
+    "type": "gitlab",
+    "baseUrl": "https://my-gitlab-host",
+    "storage": {
+      "type": "projects",
+      "path": "clients/mia-platform/configurations"
+    }
+  }
+  ```
+
+- **environments** (_required_):
+  
+  An array of objects containing the definitions of the environments for the company.
+
+  :::note
+  The content of this array will be interpolated during project creation by replacing `%projectId%` strings with the `projectId` field specified in the project.
+  :::
+
+  Each object is shaped as follows:
+
+   - **label**: descriptive and human-readable identifier associated with the environment;  
+   - **envId**: unique identifier associated with the environment;
+   - **envPrefix**: defines a common prefix associated with environment-specific variables;
+   - **description**: short description of the environment;
+   - **isProduction**: boolean value indicating whether it is considered to be a production environment;
+   - **hosts**: TODO
```

Comment:
An array of object, which specify at the project creation which host will be exposed (if exposed in template), used by links in environment sections

Something like:

```json
    "hosts": [
      {
        "host": "test.%projectId%.console.gcp.mia-platform.eu",
        "isBackoffice": false,
        "scheme": "https"
      }
    ],
```

Where isBackoffice and scheme are optional.